src/restaurants/restaurants.service.ts

```diff
@@ -1,8 +1,41 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Restaurant } from './entities/restaurants.entity';
+import { CreateRestaurantDto } from './dto/create-restaurants.dto';
+import { UpdateRestaurantDto } from './dto/update-restaurants.dto';
 
 @Injectable()
 export class RestaurantsService {
-  getRestaurants(): string {
-    return 'RestaurantsService 111';
+  constructor(
+    @InjectRepository(Restaurant)
+    private readonly restaurantRepository: Repository<Restaurant>,
+  ) {}
+
+  async getRestaurantById(restaurantId: string): Promise<Restaurant> {
+    const restaurant = await this.restaurantRepository.findOne({ where: { id: restaurantId } });
+    if (!restaurant) {
+      throw new NotFoundException(`Restaurant with ID ${restaurantId} not found`);
+    }
+    return restaurant;
```

Comment:
return문 전에는 개행을 해주시는 습관을 들여보시면 좋을 것 같아요~
```suggestion
    const restaurant = await this.restaurantRepository.findOne({ where: { id: restaurantId } });
    if (!restaurant) {
      throw new NotFoundException(`Restaurant with ID ${restaurantId} not found`);
    }
    
    return restaurant;
```